htdocs/resource/class/dolresource.class.php

```diff
@@ -51,6 +51,11 @@ class Dolresource extends CommonObject
 	public $busy;
 	public $mandatory;
 
+	/**
+	 * @var integer
+	 */
+	public $socid;
```

Comment:
If i look at the record of a dolresource, i don't see field company (socid).
So i don't think the problem is that the declaration is missing. Problem may be on the other side where the field is used and where it should not.

![image](https://user-images.githubusercontent.com/883887/204667213-db877fb8-2a9f-4e66-834a-f094d0a72e3e.png)